jekyll/_cci2/basics.md

```diff
@@ -9,12 +9,13 @@ Get started by learning about the basic concepts used in CircleCI.
 
 Document | Description
 ----|----------
-<a href="{{ site.baseurl }}/2.0/about-circleci/">Overview</a> | Overview of Continuous Integration (CI) with links to CircleCI case studies.
+[Overview]({{ site.baseurl }}/2.0/about-circleci/) | Overview of Continuous Integration (CI) with links to CircleCI case studies.
 [Using Containers]({{ site.baseurl }}/2.0/containers/) | An overview of containers and how to use them to increase build speed and prevent queuing.
 [Concepts]({{ site.baseurl }}/2.0/concepts/) | A high-level overview of CircleCI 2.0 Steps, Image, Jobs, and Workflows concepts and the configuration hierarchy.
-<a href="{{ site.baseurl }}/2.0/jobs-steps/">Jobs and Steps</a> | How Jobs and Steps are used in a CircleCI 2.0 configuration.
-<a href="{{ site.baseurl }}/2.0/workflows/">Workflows</a> | How to configure Workflows to increase the speed of your software development through faster feedback, shorter reruns, and more efficient use of resources.
-<a href="{{ site.baseurl }}/2.0/faq/">FAQ</a> | Frequently asked questions about CircleCI 2.0.
+[Jobs and Steps]({{ site.baseurl }}/2.0/jobs-steps/) | How Jobs and Steps are used in a CircleCI 2.0 configuration.
```

Comment:
thank you! I'm not sure if this is Basics or Test...

Reply:
I don't think "Test" is a great fit. Shell scripts could be used beyond the testing phase of a build.

I do think it belongs somewhere on the "Basics" landing page, but I'm unsure of which group it belongs to there (General, Migration, or Features). I think "General" is probably the best fit for now.